### DISCUSSION TITLE
How to setup cloud sync

### DISCUSSION THREAD

**Discussion by zany130** [vikdevelop/SaveDesktop#324] [Q&A]
![image](https://github.com/user-attachments/assets/94b76f78-3ce1-41d1-ba0f-3565cd252c58)

I clicked on connect to the cloud storage, set it up, and nothing syncs?

went into setup sync file like the instructions say and I got this

![image](https://github.com/user-attachments/assets/91ba1d2a-08b3-46d5-af02-905029c77749)

I'm really confused about how to set up a sync between two PCs, and the old sync options seem to be gone.

I tried clicking the manual sync button in the hamburger menu and nothing happened.

I checked my cloud folder and its empty

**Comment by zany130** [comment]
So I figured it out.

first, I didn't realize there was documentation  here https://vikdevelop.github.io/SaveDesktop/wiki/synchronization


but anyway, I had to delete the flatpacks user data in `~/.var/app/io.github.vikdevelop.SaveDesktop`  and then set everything up again according to the above documentation

**Comment by vikdevelop** [comment]
I apologize for the late reply. It's great that you've already solved it!
